core/src/main/java/tc/oc/pgm/regions/RegionMatchModule.java

```diff
@@ -128,7 +125,6 @@ public void applyEffects(final PlayerCoarseMoveEvent event) {
 
     Vector from = event.getBlockFrom().toVector();
     Vector to = event.getBlockTo().toVector();
-    Query query = new tc.oc.pgm.filters.query.PlayerQuery(event, player);
```

Comment:
By getting rid of this, and using just the player instead of the query, you're breaking any filter that relies on the event, (eg: cause filters)